### DISCUSSION TITLE
Pregunta configuración archivo launch.json Windows (WSL)

### DISCUSSION THREAD

**Discussion by IanZmen** [IIC2133-PUC/2024-1#4] [Ayuda] [answered]
Estimados, estaba tratando de realizar la configuración de mi debugger en visual, siguiendo la configuración de https://github.com/IIC2133-PUC/2024-1/blob/main/Gu%C3%ADas/4.%20Gu%C3%ADa%20para%20debuggear%20en%20C.md se ve reflejada en mi archivo como:

![image](https://github.com/IIC2133-PUC/2024-1/assets/105300405/ac8da860-7000-4cc0-8c03-cfb54830f1c6)

pero, me aparece el siguiente error:

![image](https://github.com/IIC2133-PUC/2024-1/assets/105300405/1891d643-58cf-4bef-81ae-00123b674a6e)

que debería hacer para que funcione?

Solución: El debugger esta solo en ubuntu por lo que tenía que trabajar encima de este y no directamente en mis archivos.

**Comment by benjavicente** [comment, marked as answer]
> Solución: El debugger esta solo en ubuntu por lo que tenía que trabajar encima de este y no directamente en mis archivos.

Voy a marcar la pregunta como solucionada con esto 😅
